src/linux/process.rs

```diff
@@ -457,7 +457,7 @@ fn update_time_and_memory(
             entry.memory -= parent_memory;
         }
         // vsz
-        entry.virtual_memory = u64::from_str(parts[22]).unwrap_or(0);
+        entry.virtual_memory = u64::from_str(parts[22]).unwrap_or(0) * info.page_size_kb;
```

Comment:
From the man page:

>              (23) vsize  %lu
>                     Virtual memory size in bytes.
>
>              (24) rss  %ld
>                     Resident Set Size: number of pages the process has in real memory.  This is just
>                     the pages which count toward text, data, or stack space.  This does not  include
>                     pages  which  have  not  been  demand-loaded in, or which are swapped out.  This
>                     value is inaccurate; see /proc/[pid]/statm below.

So in this case, it should be a division, not a multiplication (just like the PR title states).

And also, it shouldn't divide with `info.page_size_kb` but simply by `1_000`:

```suggestion
        entry.virtual_memory = u64::from_str(parts[22]).unwrap_or(0) / 1_000;
```

Reply:
Oh ok sorry. Will fix. Thanks! Thought it was a typo.